templates/stackage-diff.hamlet

```diff
@@ -1,5 +1,10 @@
 <h1>Compare Stackage snapshots
 
+$if prevprevSnapName /= name1
+  <div .snapshot-nav>
+    <a href=@{StackageDiffR prevprevSnapName name1}>
+      \< previous diff
```

Comment:
Are you trying to escape the `<` ? Does this work? I guess we'll find out.

Reply:
Yes it works, since it is already used in snapshots-nav.hamlet